### DISCUSSION TITLE
How to stop the running of payloads when the 404 status code is matched

### DISCUSSION THREAD

**Discussion by M00nBack** [projectdiscovery/nuclei#3009] [Q&A] [answered]
I have such a requirement. For example, I am running a burst template, requesting the/user/login route. In fact, some targets do not have the/user/login route. Therefore, I want to stop immediately after matching the 404 status code. How to do it

**Comment by MetzinAround** [comment, marked as answer]
A workflow might be the way to get this done. A workflow runs templates in order and can be conditional. 

https://nuclei.projectdiscovery.io/templating-guide/workflows/

**Reply by MetzinAround** [reply]
A matcher named condition check might work, too. Same doc. 

https://nuclei.projectdiscovery.io/templating-guide/workflows/
